lib/node_modules/@stdlib/lapack/base/dgttrf/test/fixtures/large_positive_stride_1.json

```diff
@@ -0,0 +1,30 @@
+{
+  "N": 3,
+
```

Comment:
I suggest adding `A` matrix here just for reference and to help future readers grok the intent.